.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,23 @@
+name: Smoke test
+
+on:
+  schedule:
+    - cron: '0 0 * * SUN'
+  workflow_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 14
+    - run: |
+        npm install
+        npm link
+        npm link eslint-plugin-react
```

Comment:
i'm a bit confused why this is needed - `npm link` alone ensures that the binary is globally available

Reply:
ESLint is unable load the plugin without this. As far as I understand, the `npm link <package>` creates the actual symlink under projects `node_modules`. Does this mean ESLint is unable to use globally available packages?

Here is example run without this line. https://github.com/AriPerkkio/eslint-plugin-react/runs/2477044926?check_suite_focus=true

```
Error: Configuration validation errors:
- eslintrc: Failed to load plugin 'react' declared in 'CLIOptions': Cannot find module 'eslint-plugin-react'
```

Reply:
ahhh yes, because eslint `require`s it.

in that case, an alternative to "link to global, then link back down" is if we add to devDeps `"eslint-plugin-react": "link:./"`, perhaps?

Reply:
`"link:./"` seems to fail but `"file:./"` works. No need to run link steps in workflow now. https://docs.npmjs.com/cli/v7/configuring-npm/package-json#local-paths

```
"eslint-plugin-react": "file:./",
```

Reply:
Oh actually I'll need to revert this change. Some older node versions seem to fail. Some fail in unit-test step, some fail in installation step.

https://github.com/yannickcr/eslint-plugin-react/actions/runs/803786800

Reply:
seems it behaves differently in npm 6 vs 7 :-/